source/navigator/CustomStackNavigator.tsx

```diff
@@ -99,14 +103,18 @@ const CustomStackNavigator = ({
   };
 
   const NavigatorContextComponent = (
-    <NavigationHelpersContext.Provider key="navigationHelpersContext" value={navigation}>
+    <NavigationHelpersContext.Provider
+      key="navigationHelpersContext"
+      value={navigation}
+    >
       <FlexWrapper {...panResponder.panHandlers} style={[contentStyle]}>
         {descriptors[state.routes[state.index].key].render()}
       </FlexWrapper>
     </NavigationHelpersContext.Provider>
   );
 
-  const showInactivityModal = !isActive && isAuthenticated;
+  const showInactivityModal =
+    !isActive && userAuthState === USER_AUTH_STATE.SIGNED_IN;
```

Comment:
I think 'showInactivityModal' is not telling me what kind of check is evaluated.
How is active, user?
Maybe it could be somewhat more telling in what is checked like: isUserActiveAndSignedIn?

Reply:
will change constant name